test/language/statements/class/elements/private-getter-brand-check-multiple-evaluations-of-class-realm.js

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2019 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Every new evaluation of a class creates a different brand (private getter)
+esid: sec-privatefieldget
+info: |
+  ClassTail : ClassHeritage { ClassBody }
+    ...
+    11. Let proto be ObjectCreate(protoParent).
+    ...
+    31. If PrivateBoundIdentifiers of ClassBody contains a Private Name P such that the P's [[Kind]] field is either "method" or "accessor",
+      a. Set F.[[PrivateBrand]] to proto.
+    ...
+
+  PrivateBrandCheck(O, P)
+    1. If O.[[PrivateBrands]] does not contain an entry e such that SameValue(e, P.[[Brand]]) is true,
+      a. Throw a TypeError exception.
+features: [class, class-methods-private]
+flags: [noStrict]
+---*/
+
+let eval1 = $262.createRealm().global.eval;
+let eval2 = $262.createRealm().global.eval;
+
+let classStringExpression = `(
+class {
+    get #m() { return 'test262'; }
+
+    access(o) {
+      return o.#m;
+    }
+  }
+)`;
+
+let createAndInstantiateClass = function (eval) {
+  let C = eval(classStringExpression);
+  let c = new C();
+  return c;
```

Comment:
what happens here if:

1. c1 and c2 are done both with `eval1`? (or just the same reference to eval)
2. you use a direct eval:  `let C = $262.createRealm().global.eval(classStringExpression);`?
3. use a local eval, direct and indirect?

Those might be different purposes but for n. 1, I'm not sure it will differentiate from the current test status. So you might only need one reference to eval instead of `eval1` and `eval2`. WDYT?

Reply:
These questions extends to the part you verify the access method as well.

Reply:
This is interesting, if we forsake the different realms aspect of this test, and use only one eval, it should still work. it would be a different form of the class factory stuff @caiolima wrote.

If I do that, I'll probably use eval from the same context instead of another realm.. thoughts @leobalter ?

Reply:
Based on your comments I can leave the realm stuff and address the other issues